test/unit/index.ts

```diff
@@ -5,6 +5,21 @@ import fc from "fast-check"
 
 describe("index", () => {
   describe("create", () => {
+    describe("constructors", () => {
+      it("are reference-equal", () => {
```

Comment:
Should we also test that the constructor return values (for non-nullary constructors) are equal by value?